pom.xml

```diff
@@ -126,7 +156,7 @@
 		<dependency>
```

Comment:
Are you able to remove this dependency? It should be coming through via spring-cloud-aws-core?

Reply:
I will verify whether this is necessary now.

Reply:
Verified that it still works, removed.